docs/test-fairy/acct-mgmt/sso/sso-intro.md

```diff
@@ -0,0 +1,24 @@
+---
+id: sso-intro
```

Comment:
Renamed file from "sso" to "sso-intro" because there's another file called "sso.md"